templates/killmii.html

```diff
@@ -0,0 +1,27 @@
+<html>
+     <body>
+        <p>
+           {{ form.miiid.label }}<br>
+           {{ form.miiid(size=32) }}<br>
+           {% for error in form.miiid.errors %}
+           <span class="warning">[{{ error }}]
+         </p>
+        <p>{{ form.submit() }}</p>
+    </form>
+</body>
+<footer>
+<script src="https://yuzu.api.wiilink24.com/lib5.js">
+</script> 
+<script>
+window.onload=func();
+function func() {
+  $.ajax({
+    url: 'https://miisco.wii.wiilink24.com/killmii.php',
+    success: function(data) {
+      $('.result').html(data);
+    };
+  });
+};
+</script>
```

Comment:
Similarly here.. this doesn't seem functional, and the entire killmii page is implemented within the main Concierge template.